packages/stargate/src/queries/queryclient.ts

```diff
@@ -234,7 +234,7 @@ export class QueryClient {
   // throws an error if height is 0 or undefined
   private async getNextHeader(height?: number): Promise<Header> {
     assert(height);
-    if (height == 0) {
+    if (height === 0) {
```

Comment:
The `assert` already guarantees this is not 0. We could use `assertDefined(height);` instead to keep the specific error message for 0.